docs/home/contribute/includes.md

```diff
@@ -82,7 +136,7 @@ kubectl apply -f "https://git.io/weave-kube"
 {{ "{% include tabs.md " }}%}
 ````
 
-### Capturing tab content
+#### Capturing tab content
```

Comment:
For clarity, could you add a quick sentence to the end of the previous section ("Main tab code") explaining the following sections? e.g.

> The following sections break down into detail how this code works.

Reply:
:white_check_mark: